quicklogic_fasm/pp3/ql725a.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python3
```

Comment:
Would it be possible to further deduplicate `ql732b` and `ql725a` code? This and ql732b's classes have a lot in common.

Reply:
I modified code for `ql732b` and `ql725a` to be simple wrappers for the `QLAssembler` class.
At this moment I'm not entirely sure about the modifications that will be required to add proper bitstream generation support for PP3 and this is the reason why I left the `ql732b` and `ql725a` classes instead of just instantiating the `QLAssembler` with different parameters straight in `qlfasm.py`. Those classes will probably be modified in next PRs when the bitstream support will be developed.

Reply:
Ok, that makes sense.